FileManager/App/SceneDelegate.swift

```diff
@@ -22,6 +22,9 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
 
         window?.rootViewController = navigationController
         window?.makeKeyAndVisible()
+        
+        AuthorizationService.shared.delegate = viewController
```

Comment:
Shouldn't you set `navigationController` as `delegate` here because it's a root vc?